packages/app-headless-cms/src/admin/plugins/fieldRenderers/file/SingleFile.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import { makeRenderImagePreview } from "./utils";
+import SingleImageUpload from "@webiny/app-admin/components/SingleImageUpload";
+import fileIcon from "../../fields/icons/round_insert_drive_file-24px.svg";
+
+const imageExtensions = [".jpg", ".jpeg", ".gif", ".png", ".svg"];
```

Comment:
BTW, `FileManager` accepts `images` prop, so you don't have to specify file extensions by yourself.

`packages/app-admin/src/components/FileManager.tsx:73`

Reply:
BTW now I see why you needed this, for the lower `isImage` check.

At least, you got that tip.

Probably you've seen it already anyways.